servicetalk-examples/grpc/protoc-options/src/main/java/io/servicetalk/examples/grpc/protocoptions/BlockingProtocOptionsServer.java

```diff
@@ -23,8 +25,12 @@
 public final class BlockingProtocOptionsServer {
     public static void main(String[] args) throws Exception {
         GrpcServers.forPort(8080)
-                .listenAndAwait((BlockingGreeterService) (ctx, request) ->
-                        HelloReply.newBuilder().setMessage("Hello " + request.getName()).build())
+                .listenAndAwait(new BlockingGreeterService() {
```

Comment:
Was this change required because `BlockingGreeterService` is no longer a `@FunctionalInterface`?

Reply:
That is correct, in fact it was never officially an enforced `@FunctionalInterface`